docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.5'
+
+services:
+
+  vegeta:
+    build: .
+    image: flaviostutz/vegeta
+    ports:
+      - 8880:8880
+    environment: 
+      - TARGET_URL=https://www.uol.com.br
```

Comment:
Can we change this default?

Reply:
Sure. I am changing to www.yahoo.com, is this ok?